sql/1dbFamily.sql

```diff
@@ -1,9 +1,16 @@
+DROP TABLE IF EXISTS `dbFamily_languages`;
+DROP TABLE IF EXISTS `dbFamily_Assistance`;
+DROP TABLE IF EXISTS `dbLanguages`;
+DROP TABLE IF EXISTS `dbAssistance`;
+DROP TABLE IF EXISTS `dbFamily`;
```

Comment:
You'll have to drop some of the foreign key relations to get this to work and then readd them
![image](https://github.com/user-attachments/assets/70e0b204-f4cc-4ca4-aa50-7a2bfdcca4f8)